app/assets/stylesheets/mixins.scss

```diff
@@ -148,3 +148,18 @@
     transition: none;
   }
 }
+
+@mixin has-fa-icon($icon, $style) {
+  @extend .fa-#{$icon};
```

Comment:
Unfortunately fontawesome doesn't provide placeholder selectors, and we can't use the `$icon` variable to write something like:

```scss
  ::before {
    content: $fa-var-#{$icon}
  }
```

Because it isn't valid SCSS.